index.html

```diff
@@ -342,23 +353,34 @@
   </section>
   <section id="sec-beacon">
     <h2>Beacon</h2>
-    <section id="sec-sendBeacon-method">
-      <h2><code>sendBeacon</code> Method</h2>
+    <section id="sec-beacon-methods">
+      <h2>Methods</h2>
       <pre class="idl">
-partial interface <dfn id="Navigator">Navigator</dfn> {
-    boolean <a>sendBeacon</a>(<a>USVString</a> <var title=
-"url">url</var>, optional <a>BodyInit</a>? <var>data</var> = null);
+partial interface Navigator {
+    boolean sendBeacon(USVString url, optional BodyInit? data = null);
 };
 
-partial interface <dfn id="WorkerNavigator">WorkerNavigator</dfn> {
-    boolean <a>sendBeacon</a>(USVString <var title=
-"url">url</var>, optional <a>BodyInit</a>? <var>data</var> = null);
+partial interface WorkerNavigator {
+    boolean sendBeacon(USVString url, optional BodyInit? data = null);
 };
-</pre>
-      <p>The <dfn id="dom-navigator-sendbeacon"><code>sendBeacon</code></dfn>
-      method transmits data provided by the <a href=
-      "#data-parameter"><code>data</code></a> parameter to the URL provided by
-      the <a href="#url-parameter"><code>url</code></a> parameter:</p>
+
+[NoInterfaceObject, Exposed=(Window,Worker)]
+partial interface GlobalFetch {
```

Comment:
This should now become WindowOrWorkerGlobalScope per a recent update to Fetch and HTML.